### ISSUE TITLE
Support for .graphqlconfig in a subdirectory

### ISSUE THREAD

**Issue by BenDiuguid** [prisma/vscode-graphql#24]
Hey this extension is awesome!! 😄 

I would love to see some support for loading `.graphqlconfig` files from either subdirectories, or the setting/ability to load from a specified file/folder.

My set up looks like the following:

```
project_root/
    client/
        ...
    server/
        .graphqlconfig.yml
        ...
```

It seems like [I'm not the only one](https://github.com/prismagraphql/vscode-graphql/issues/7#issuecomment-397913252) experiencing this.

I'm currently receiving this error: 
<img width="489" alt="screen shot 2018-06-18 at 5 58 46 pm" src="https://user-images.githubusercontent.com/7402211/41571658-09bdc164-7329-11e8-9c46-7561bfc0a7d2.png">

**Comment by airato** [comment]
related to this - need support for projects that use [lerna](https://github.com/lerna/lerna) and have multiple npm packages with their own tsconfig and schema.graphql files. Need ability to add separate .graphqlconfig files for those packages, and the autocomplete and jump to definition should respect the scope.

**Comment by ry5n** [comment]
I ran into this using the full stack react boilerplate from [graphql-boilerplates](https://github.com/graphql-boilerplates/react-fullstack-graphql/tree/master/advanced). It has a `/server` subdirectory with its own `package.json` and `.graphqlconfig.yml`  😢

**Comment by divyenduz** [comment]
This is released in 0.1.4 (which is a preview release with CodeLens feature as well) 🎉 

Feedback please 👍👇 
![](http://g.recordit.co/KMsQhNflnF.gif)

**Comment by ry5n** [comment]
Amazing! Thank you 🎉
